gensim/corpora/linesentence.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2013 Zygmunt Zając <zygmunt@fastml.com>
```

Comment:
@piskvorky Are these copyright statements legit?

Reply:
I have taken it from [here](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/corpora/csvcorpus.py) .

Reply:
But, even if it was accurate there, this code is from many places & many authors, so it's not appropriate to attribute it all to another person in past time. 

In a few cases where many have contributed, I've changed the leading comment to read like (but it should probably say '2020' upon updates):

https://github.com/RaRe-Technologies/gensim/blob/683cebef27a0bf5e9af39d9b5b0a522e62f20862/gensim/models/doc2vec.py#L4-L5

Still, it'd be good to have a project standard for how authorship/copyright is declared per file. Saying less makes more sense than saying more that isn't true.

Reply:
Agreed with @gojomo . I plan to clarify the developer instructions as part of #2948.

TL;DR: authorship cannot be transferred in many legislations (including EU), it's for life. In order for us (maintainers) to be able to publish & modify the code, we need full assignment of all IP rights from the contributor. Which is what we've been [implicitly doing](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#legal) but perhaps should make more explicit.

Reply:
While I've seen more complicated processes, and the State of the Art may have advanced since I last looked, my non-lawyerly hunch is that a low-effort system could be: (1) There's a very-simple rights-assignment statement in the repo; (2) To accept someone's contributions, they must 1st (either before or in their 1st PR) include the addition of their Github username to a list of developers granting such assent, at the bottom of that statement. Thus the same mechanism of contribution collects overt/authenticated expressions of assignment, while building a master-list of contributors as a side-effect.

Reply:
Sorry I did not know about these things. 
I have changed it to 
``` 
 # Author: Gensim Contributors 
 # Copyright (C) 2020 RaRe Technologies s.r.o.
```
Sorry again.